hadoop-ozone/ozone-manager/dev-support/findbugsExcludeFile.xml

```diff
@@ -89,4 +89,9 @@
     <Class name="org.apache.hadoop.ozone.om.upgrade.TestOMLayoutFeatureAspect"/>
     <Bug pattern="BC_IMPOSSIBLE_CAST" />
   </Match>
+  <Match>
+    <Class name="org.apache.hadoop.ozone.om.ratis.OzoneManagerDoubleBuffer" />
+    <Field name="readyBuffer" />
+    <Bug pattern="IS2_INCONSISTENT_SYNC" />
+  </Match>
```

Comment:
@swamirishi , How about adding `synchronized` to `splitReadyBufferAtCreateSnapshot()`?

Reply:
We don't need an object lock on double buffer. That would impact the entire write flow. Only flushing daemon uses this readBuffer. Active writes happen to current buffer we shouldn't be blocking that. Object lock will block that. Ideally we could have implemented individual locks than depending on the entire object lock. This would be more optimized.

Reply:
The `splitReadyBufferAtCreateSnapshot()` just has a simple loop.  Adding `synchronized ` won't affect the performance   On the other hand, suppressing `IS2_INCONSISTENT_SYNC` will hide the real bug in the future.

Reply:
yeah I agree about that. Ok then let me make the method synchronized